src/ts/MemorisedBar.svelte

```diff
@@ -79,8 +89,30 @@
     }
 </script>
 
-{#if retrievabilityDays}
-    <LineOrCandlestick data={retrievabilityDays} label={i18n("cards")} bind:trend_data />
+{#if retrievabilityDays && stable_retrievability_days}
+    <div>
+        <label>
+            <input type="radio" value={MemorisedType.RETRIEVABILITY} bind:group={memorised_type} />
+            {i18n("Retrievability")}
+        </label>
+        <label>
+            <input
+                type="radio"
+                value={MemorisedType.STABLE_RETRIEVABILITY}
+                bind:group={memorised_type}
+            />
+            {i18n("Retrievability and (Sqrt) Stability)")}
```

Comment:
```suggestion
            {i18n("retrievability-times-sqrt-stability)")}
```
You need to add this in https://github.com/Luc-Mcgrady/Anki-Search-Stats-Extended/blob/main/locale/en_GB.ftl